<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>362</width>
    <height>488</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="save_skins_button">
       <property name="text">
        <string>Save Skins</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="save_ctrls_button">
       <property name="text">
        <string>Save Ctrls</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="save_guide_button">
       <property name="text">
        <string>Save Guide</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="load_skins_button">
       <property name="text">
        <string>Load Skins</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="load_ctrls_button">
       <property name="text">
        <string>Load Ctrls</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="load_guide_button">
       <property name="text">
        <string>Load Guide</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="3" column="0">
    <widget class="QScrollArea" name="scrollArea">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>342</width>
        <height>365</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <layout class="QVBoxLayout" name="verticalLayout"/>
       </item>
       <item row="1" column="0">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QComboBox" name="files_combo_box"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
